### ISSUE TITLE
resdcn18 on VOC

### ISSUE THREAD

**Issue by zjp99** [xingyizhou/CenterNet#501]
I train 70 epoch and drop lr 45,60.but test result on VOC,mAp= 67.51(384)//67.48(512),what should I do?

**Comment by xingyizhou** [comment]
Can you provide your commands for both training and testing?

**Comment by zjp99** [comment]
@xingyizhou
 # train（train8218+val8333）
python main.py ctdet --exp_id pascal_resdcn18_384 --arch resdcn_18 --dataset pascal --num_epochs 70 --lr_step 45,60 --flip_test --batch_size 16 
# flip test（4932）
python test.py ctdet --exp_id pascal_resdcn18_384 --arch resdcn_18 --dataset pascal --resume --flip_test --trainval

**Comment by xingyizhou** [comment]
Can you try removing `--batch_size 16` as is in our [experiments](https://github.com/xingyizhou/CenterNet/blob/master/experiments/ctdet_pascal_resdcn18_384.sh_?

**Comment by zjp99** [comment]
> 
> 
> Can you try removing `--batch_size 16` as is in our [experiments](https://github.com/xingyizhou/CenterNet/blob/master/experiments/ctdet_pascal_resdcn18_384.sh_?

Sorry ，I wrote it wrong. --batch_size 32

**Comment by zjp99** [comment]
That's the command I'm using.But can't achieve your result, do I need to adjust other parameters?res18 and test on VOC.发自我的华为手机-------- 原始邮件 --------发件人： Xingyi Zhou <notifications@github.com>日期： 2019年12月4日周三 凌晨1:15收件人： xingyizhou/CenterNet <CenterNet@noreply.github.com>抄送： zjp99 <3238816982@qq.com>, Author <author@noreply.github.com>主    题： Re: [xingyizhou/CenterNet] resdcn18 on VOC (#501)Can you try removing --batch_size 16 as is in our [experiments](https://github.com/xingyizhou/CenterNet/blob/master/experiments/ctdet_pascal_resdcn18_384.sh_?

**Comment by zjp99** [comment]
@xingyizhou Do you have some  suggestions？

**Comment by xingyizhou** [comment]
I have no suggestions. Can you make sure you haven't changed any code, have disabled cudnn batchnorm, and run the exact command in https://github.com/xingyizhou/CenterNet/blob/master/experiments/ctdet_pascal_resdcn18_384.sh?

**Comment by zjp99** [comment]
@xingyizhou I will try it again.

**Comment by developer0hye** [comment]
@zjp99 
Do you solve this issue?
